sdk/communication/Azure.Communication.JobRouter/src/RouterClient.cs

```diff
@@ -346,6 +346,59 @@ protected RouterClient()
             }
         }
 
+        /// <summary> Protocol method to use to remove properties from job. </summary>
+        /// <param name="jobId"> Id of the job. </param>
+        /// <param name="patchContent"> Request content payload. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        /// <exception cref="RequestFailedException">The server returned an error. See <see cref="Exception.Message"/> for details returned from the server.</exception>
+        public virtual async Task<Response<RouterJob>> UpdateJobAsync(
```

Comment:
Protocol methods should return `Response` and take `RequestContent` and `RequestContext` parameters - you can see the [protocol method docs](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/samples/ProtocolMethods.md) for reference.